wheatley/aliases.py

```diff
@@ -1,10 +1,11 @@
 """ A module to hold common type aliases """
 
-from typing import Any, Dict, NewType, List
+from typing import Any, Dict, NewType, List, Tuple, Optional
 from wheatley.bell import Bell
 
 
 CallDef = NewType('CallDef', Dict[int, str])
 JSON = Dict[str, Any]
 Places = NewType('Places', List[int])
 Row = NewType('Row', List[Bell])
+RowCall = Tuple[Row, Optional[str]]
```

Comment:
I reckon this could do with a better name, but I don't want to make the name unnecessarily long.  Also it to change it's type to `(Row, [str])` (Haskell-style syntax) in case multiple calls are placed on the same row.

Maybe something like `RowAndCalls` or `RowWithCalls` ... what do you reckon?

Reply:
I would consider not introducing a type and just keep it as `Tuple[Row, List[str]]` on the method return as needed. Maybe a type for `SpokenCalls = List[str]`

Reply:
I think that is a good idea, if we keep going with the current Rust-iterator-style API (for wont of a better name).  It's a pity that Python's type signatures are so verbose - `(Row, [str])` is so much easier to read than `Tuple[Row, List[str]]`.  But it is what it is.